graylog2-server/src/main/java/org/graylog2/inputs/transports/KafkaTransport.java

```diff
@@ -389,8 +575,18 @@ public ConfigurationRequest getRequestedConfiguration() {
                     DEFAULT_GROUP_ID,
                     "Name of the consumer group the Kafka input belongs to",
                     ConfigurationField.Optional.OPTIONAL));
+            cr.addField(new TextField(
+                    CK_CUSTOM_PROPERTIES,
+                    "Custom Kafka properties",
+                    "",
+                    "A newline separated list of Kafka properties. (e.g.: \"ssl.keystore.location=/etc/graylog/server/kafka.keystore.jks\").",
+                    ConfigurationField.Optional.OPTIONAL,
+                    110,
```

Comment:
Instead of a magic number we should use a constant introduced in `ConfigurationField` here, maybe something like `BELOW_DEFAULT`?

Reply:
ack!